lib/hammer_cli_foreman/auth.rb

```diff
@@ -1,34 +1,69 @@
-module HammerCLIForeman
+require 'hammer_cli_foreman/keycloak.rb'
```

Comment:
When we decided to make the integration for generic OAuth shouldn't we get rid of Keycloak being mentioned in the code? Do we have any Keycloak specific code in here or we can rename it to OAuth across the codebase?